api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -54,11 +54,31 @@ message UpstreamTlsContext {
   google.protobuf.UInt32Value max_session_keys = 4;
 }
 
-// [#next-free-field: 8]
+// [#next-free-field: 9]
 message DownstreamTlsContext {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.api.v2.auth.DownstreamTlsContext";
 
+  enum OcspStaplePolicy {
+    // OCSP responses are optional. Invalid or expired OCSP
+    // responses are rejected at config time. If an OCSP
+    // response expires before a config update, it is no
+    // longer stapled.
+    SKIP_STAPLING_IF_EXPIRED = 0;
```

Comment:
What's the behavior if no staple is configured at all? Please clarify docs for that case.